test/unit/manipulation.js

```diff
@@ -3099,3 +3099,55 @@ testIframe(
 		} );
 	}
 );
+
+// Custom event handler test case
+
+QUnit.test( "should handle custom '_se-custom-destroy' event correctly", function( assert ) {
```

Comment:
This is not about any specific event; also, this wording doesn't follow the style from other tests. How about:
```suggestion
QUnit.test( "Cleans data for events removing elements on `remove` (gh-5214)", function( assert ) {
```
Maybe others have a better suggestion for a more succinct name.

Reply:
Done